.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+name: build
+
+on:
+  push:
```

Comment:
Should we do this?
on: [push, pull_request]
and not specify the branch as master? 
so it will run on any new PR created? WDYT?

Reply:
Done

Reply:
Did you push this? apologies, don't see it.

Reply:
Sorry, changed, but did not push the changes.